decidim-proposals/app/commands/decidim/proposals/admin/reply_proposal_note.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Proposals
+    module Admin
+      # A command with all the business logic when an admin creates a private note proposal reply.
+      class ReplyProposalNote < Decidim::Command
+        include ProposalNotesMethods
+
+        # Public: Initializes the command.
+        #
+        # form   - A form object with the params.
+        # parent - the note to reply.
+        def initialize(form, parent)
+          @form = form
+          @parent = parent
+          @proposal = parent.proposal
+        end
+
+        # Executes the command. Broadcasts these events:
+        #
+        # - :ok when everything is valid, together with the note proposal.
+        # - :invalid if the form was not valid and we could not proceed.
+        #
+        # Returns nothing.
+        def call
+          return broadcast(:invalid) if form.invalid? || invalid_parent?
+
+          create_proposal_note_reply
+          notify_mentioned_or_parent_author
+
+          broadcast(:ok, proposal_note)
+        end
+
+        private
+
+        attr_reader :form, :proposal_note, :parent, :proposal
+
+        def invalid_parent?
+          parent.blank? || parent.reply?
+        end
+
+        def create_proposal_note_reply
+          @proposal_note = Decidim.traceability.create!(
+            ProposalNote,
+            form.current_user,
+            {
+              body: rewritten_body,
+              proposal:,
+              parent:,
+              author: form.current_user
+            },
+            resource: {
+              title: proposal.title
+            }
+          )
+        end
+
+        def notify_mentioned_or_parent_author
```

Comment:
Same here, I think we need to have two different events:

1. Notification to the parent author of the private note: This should be like it is for `reply_created` in comments event: "`YYY @zzz has replied your comment in XXXX`", so in this case it should be "`YYY @zzz has replied your private note in XXXX`"
2. Notification to the valuator/administrator that was mentioned: `You have been mentioned in a private note in XXX by YYY @zzz. Check it out at the admin panel.`

Reply:
Added 2 different events texts for the same event which includes the author of the note